src/icons/README.md

```diff
@@ -3,36 +3,27 @@
 
 ## Previous
 Initial `qbt-theme` icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
-
 Icons were based on the [Font-Awesome](http://fontawesome.io/icons/) icon set.
-
 ## Redesign
-
-Redesigned icons are created by Nowshed H. Imran (<now.im.627@gmail.com>).
-
+Redesigned icons are created by Nowshed H. Imran (<nowshed.imran@gmail.com>).
 New icons are based on [la-capitaine-icon-theme](https://github.com/keeferrourke/la-capitaine-icon-theme).
-
-`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them. Get an icon from there if needed. 
-
+`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them. Get an icon from there if needed.
 ## Color RGBA
-
-Here is the following folowing color RGBA used for icon uniformity.
+Here is the following color RGBA used for icon uniformity.
 
 |     Color        |     RGBA         |
 |:----------------:|:----------------:|
-|     Blue         |     4285f4ff     |
-|     Red          |     db3236ff     |
-|     Green(Deep)  |     0fbb58ff     |
-|     Green(Light) |     6cc24aff     |
-|     Grey         |     737373ff     |
-|     Orange       |     f25022ff     |
-
-
+|     Blue         |     4285F4       |
+|     Red          |     DB3236       |
+|     Green(Deep)  |     0FBB58       |
+|     Green(Light) |     6CC24A       |
+|     Grey         |     2C2C2C       |
+|     Orange       |     F25022       |
+|     Magenta      |     C051DB       |
+|     Purple       |     AF5CF9       |
 ## Icon Optimization
-
 In order to optimize the icons [svgcleaner](https://github.com/RazrFalcon/SVGCleaner) has been used.
 Submit the icon without optimization first and after that, submit the optimized one.
 It is recommended to use "Svgcleaner" as optimization tool, as it almost never breaks any icon.
-
-
-
+## Standalone Light and Dark Icon Set(To-Do)
```

Comment:
What is the purpose of this??

Reply:
Linking, a useful tool for future reference. I am not going to do it. I have chosen colors that work both light and dark theme, but not perfectly. For that, two icon packs is needed. 
[Material](https://material.io/resources/color/) is a brilliant tool for that. I can remove it if you want, but someone might implement the feature in the future. It will definitely help.

Reply:
Or I can mention this under color that I have used this tool to find colors that work for both light and dark theme.

Reply:
>Or I can mention this under color that I have used this tool to find colors that work for both light and dark theme.

This would be better, IMO

>two icon packs is needed.

IMO I don't think that is totally necessary but I don't want to get into too much of a debate on that.

Reply:
Done.